---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---
## 2026


* [The interpretable multimodal dimension reduction framework SpaHDmap enhances resolution in spatial transcriptomics](https://www.nature.com/articles/s41556-025-01838-z)<br>
  Junjie Tang\*, Zihao Chen\*, **Kun Qian**\*, Siyuan Huang, Yang He, Shenyi Yin, Xinyu He, Buqing Ye, Yan Zhuang, Hongxue Meng, Jianzhong Jeff Xi, Ruibin Xi<br>
  *bioRxiv*, 2024
  
  We introduce SpaHDmap, an interpretable dimension reduction framework that enhances spatial resolution by integrating ST gene expression with high-resolution histology images. SpaHDmap incorporates non-negative matrix factorization into a multimodal fusion encoder-decoder architecture, enabling the identification of interpretable, high-resolution embeddings.

## 2025

* [A targeted DNA methylation method for detecting gastrointestinal cancer in circulating cell-free DNA](https://www.sciencedirect.com/science/article/pii/S2589004225026033)<br>Zhaoyu Jiang\*, Yuqing Guo\*, Yongqu Lu\*, **Kun Qian**\*, Xiaomeng Liu, Siyi Lu, Jie Ren, Fuchou Tang, Wei Fu, Lu Wen, Xin Zhou<br> *iScience*, 2025

  We developed tMCTA-seq, a cost-effective targeted DNA methylation method for simultaneous detection and differentiation of colorectal and gastric cancers. By integrating a universal nested primer design with an ensemble machine learning framework, we achieved high diagnostic performance across multiple cancer stages in clinical cfDNA samples.

## 2024
* [Structure-preserved integration of scRNA-seq data using heterogeneous graph neural network](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbae538/7833674)<br>Xun Zhang\*, **Kun Qian**\*, Hongwei Li<br>*Briefings in Bioinformatics*, 2024 

  By establishing a heterogeneous graph that represents the interactions between multiple batches of cells and genes, and combining a heterogeneous graph neural network with contrastive learning, we proposed a structure-preserved scRNA-seq data integration approach scHetG.

## 2023
* [scAce: an adaptive embedding and clustering method for single-cell gene expression data](https://doi.org/10.1093/bioinformatics/btad546)<br>
  Xinwei He\*, **Kun Qian**\*, Ziqian Wang, Shirou Zeng, Hongwei Li, Wei Vivian Li<br>
  *Bioinformatics*, 2023
  
  We propose an adaptive embedding and clustering method named scAce, which constructs a variational autoencoder to simultaneously learn cell embeddings and cluster assignments. In the scAce method, we develop an adaptive cluster merging approach which achieves improved clustering results without the need to estimate the number of clusters in advance.
  
* [Integration of scRNA-seq data by disentangled representation learning with condition domain adaptation](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-024-05706-9)<br>Renjing Liu\*, **Kun Qian**\*, Xinwei He, Hongwei Li<br>*BMC Bioinformatics*, 2023

  To expedite the exploration of systematic disparities under various biological contexts, we propose a scRNA-seq integration method called scDisco, which involves a domain-adaptive decoupling representation learning strategy for the integration of dissimilar scRNA-seq data.

* [scBiG for representation learning of single-cell gene expression data based on bipartite graph embedding](https://academic.oup.com/nargab/article/6/1/lqae004/7513703)<br>Ting Li\*, **Kun Qian**\*, Xiang Wang, Wei Vivian Li, Hongwei Li<br>*NAR Genomics and Bioinformatics*, 2023

  We introduce scBiG, a novel graph node embedding method designed for representation learning in scRNA-seq data. scBiG establishes a bipartite graph connecting cells and expressed genes, and then constructs a multilayer graph convolutional network to learn cell and gene embeddings.

## 2022
* [scINSIGHT for interpreting single-cell gene expression from biologically heterogeneous data<br>](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02649-3)**Kun Qian**\*, Shiwei Fu, Hongwei Li, Wei Vivian Li\*<br>*Genome Biology*, 2022

  Although different batch effect removal methods have been developed, none are suitable for heterogeneous single-cell samples coming from multiple biological conditions. We propose a method, scINSIGHT, to learn coordinated gene expression patterns that are common among, or specific to, different biological conditions, and identify cellular identities and processes across single-cell samples.

\* Equal contribution